vumi_http_proxy/clickme.py

```diff
@@ -3,12 +3,19 @@
 import click
 from vumi_http_proxy import http_proxy
 
+test = False
+
 
 @click.command()
 @click.option('--interface', default="0.0.0.0", help='eg 0.0.0.0')
 @click.option('--port', default=8080, help='eg 80')
 def cli(interface, port):
     """This script runs vumi-http-proxy on <interface>:<port>"""
-    i = http_proxy.Initialize(["asdf.com"], str(interface), port)
     click.echo("Starting connection to %s:%d" % (str(interface), port))
-    i.main()
+    if test is False:
+        i = http_proxy.Initialize(["asdf.com"], str(interface), port)
```

Comment:
Twisted test cases have a method we can use to help avoid this, `patch`. See https://twistedmatrix.com/documents/current/api/twisted.trial.unittest.SynchronousTestCase.html#patch for the documentation.

We can then patch out the `main` method and just check that it is called as expected.